canonical-web-and-design/maas-ui
# feat(ui): add back link to cluster settings

## Done
- Created a SettingsBackLink component which links to either virsh or lxd list
- Added the component to both settings views
- Added styling to align with [the design](https://www.figma.com/file/QNQ3ZsYLR0tamnFzCo8VP6/%5B21.10%5D-LXD-Clusters?node-id=684%3A24351)
- Added some basic tests

### QA steps
- Run this branch locally
- Visit http://localhost:8400/MAAS/r/kvm/lxd/336/edit and check the back button links to the lxd list
- Switch the Virsh list and select one
- Switch to settings
- Click the back button and see it returns you to the virsh list

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/137994114-42158ffb-1785-46bb-ad82-6e2815021afe.png)
